### ISSUE TITLE
Outdated version of iRacing

### ISSUE THREAD

**Issue by LapsTimeOFF** [Friss/iracing-sdk-js#6]
Hey!

Any plans on updating the module to [iRacing 2024 Season 3 Patch 2](https://support.iracing.com/support/solutions/articles/31000173510-2024-season-3-patch-2-release-notes-2024-06-28-01-)?

**Comment by Friss** [comment]
Is there anything in particular that isn't working with the latest version? I just checked and the current version worked just fine but I regenerated the sample data https://github.com/Friss/iracing-sdk-js/pull/7

**Comment by LapsTimeOFF** [comment]
Thanks for the regenerated sample data, but I can't get the Lat/Long data though it's mentioned in iRacing docs as available live

**Comment by Friss** [comment]
Ah from some searches of the forums it seems that the Lat/Long data is only available in the `ibt` telemetry files that are generated on delay and not in the live API data. Unless you could point me to something saying otherwise. 

- https://forums.iracing.com/discussion/comment/49121/#Comment_49121
- https://members.iracing.com/jforum/posts/list/350/1826390.page#10400569

I can look into supporting reading ibt files if that would be of interest.

**Comment by LapsTimeOFF** [comment]
I have [this post](https://forums.iracing.com/discussion/62/iracing-sdk/p1) though, in the PDF we can see that Lat and Long are available Live. If they are actually not, do you know how the track map overlays from 3rd party apps are made? Also the `.ibt` files parsing would be nice!

**Comment by Friss** [comment]
Yeah seems like that PDF is wrong based on real data and some of the discussion around the posts I linked. 

As for track maps I assume they do a process similar to this https://members.iracing.com/jforum/posts/list/3776411.page where they drive the track and take the gps data from the `.ibt` files. And then use the percent complete lap metric to draw the car along the map. 

I'll look into adding `.ibt` file support sometime soon.
